<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Orders_Add</class>
 <widget class="QWidget" name="Admin_Orders_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加或修改订单</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_bid">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>租借编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_uid">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>雨伞编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pid_b">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>借出地点编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>借出时间（unix时间戳）</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>是否归还</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pid_r">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>归还地点编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>归还时间（unix时间戳）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>447</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
